src/backend/distributed/utils/statistics_collection.c

```diff
@@ -30,17 +30,26 @@ bool EnableStatisticsCollection = true; /* send basic usage statistics to Citus
 #include "distributed/statistics_collection.h"
 #include "distributed/worker_manager.h"
 #include "lib/stringinfo.h"
+#include "utils/builtins.h"
 #include "utils/json.h"
 #include "utils/jsonb.h"
 
 #if PG_VERSION_NUM >= 100000
 #include "utils/fmgrprotos.h"
 #endif
 
+static size_t CheckForUpdatesCallback(char *contents, size_t size, size_t count,
+									  void *userData);
+static bool JsonbFieldInt(Jsonb *jsonb, const char *fieldName, int32 *result);
+static bool JsonbFieldStr(Jsonb *jsonb, const char *fieldName, StringInfo result);
 static uint64 NextPow2(uint64 n);
 static uint64 ClusterSize(List *distributedTableList);
-static bool SendHttpPostJsonRequest(const char *url, const char *postFields, long
-									timeoutSeconds);
+static bool UrlEncode(StringInfo buf, const char *str);
+static bool SendHttpPostJsonRequest(const char *url, const char *postFields,
+									long timeoutSeconds);
+static bool SendHttpGetRequest(const char *url, long timeoutSeconds,
```

Comment:
I didn't include the word `Json` in the function name since this function doesn't assume anything about response type. We could add `Accept: application/json`, but that didn't seem necessary.